tests/index.spec.js

```diff
@@ -38,12 +39,24 @@ describe(`soundslice-data-api`, () => {
     });
   });
 
+  describe(`deleteRecordingByRecordingId`, () => {
+    it(`should be a function`, () => {
+      expect(apiClient.deleteRecordingByRecordingId).to.be.a(`function`);
+    });
+  });
+
   describe(`deleteSliceBySlug`, () => {
     it(`should be a function`, () => {
       expect(apiClient.deleteSliceBySlug).to.be.a(`function`);
     });
   });
 
+  describe(`duplicateSliceByScorehash`, () => {
+    it(`should be a function`, () => {
+      expect(apiClient.duplicateSliceByScorehash).to.be.a(`function`);
+    });
+  });
+
```

Comment:
A smoke test for `duplicateSliceByScorehash` should have been added in the last PR,  https://github.com/ericcarraway/soundslice-data-api/pull/4.